examples/complete/main.tf

```diff
@@ -38,3 +38,31 @@ module "ecs_cluster" {
     }
   }
 }
+
```

Comment:
Oh nvm, I assumed this wasnt an example. I thought we were baking the asg directly in this. I like this approach better to keep the ecs cluster separate from the asg.